TokensExtraction.groovy

```diff
@@ -66,7 +66,11 @@ pipeline {
     stage('Fetch tokens') {
       steps {
         script {
-          sh "pnpm run tokens:fetch --filter @coveord/plasma-tokens"
+          if (params.LIBRARIES.length() > 0) {
+            sh "pnpm run tokens:fetch --filter @coveord/plasma-tokens -- --libraries ${params.LIBRARIES}"
```

Comment:
I remember this happening to me in the past, not sure if it's still happening https://issues.jenkins.io/browse/JENKINS-43758

One thing though is that it would be better to have the parameters in the config to be able to do PRs and modify those easily

Reply:
Since the parameter is from a custom plugin, I am not sure what to put in there. I think I could put stringParam because that's what the plugin output in the end, but I fear it could override the parameter configuration to set everything as a string param instead of the plugin

Reply:
I think it is possible, have a look at https://stackoverflow.com/a/56113578

```groovy
  extendedChoice description: 'Libraries to fetch', multiSelectDelimiter: ',', name: 'LIBRARIES', quoteValue: false, saveJSONParameterToFile: false, type: 'PT_CHECKBOX', value: 'ICONS,TEST', visibleItemCount: 50
```

Reply:
added this to the groovy script

Reply:
That's good to know if we want to create a job for the package creator :eyes: :+1: